service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/CreateAzureDatabaseStep.java

```diff
@@ -79,10 +79,9 @@ public CreateAzureDatabaseStep(
     this.kubernetesClientProvider = kubernetesClientProvider;
   }
 
-  private String getPodName(String newDbUserName) {
+  private String getPodName() {
     // strip underscores to avoid violating azure's naming conventions for pods
-    return (this.workspaceId.toString() + this.resource.getDatabaseName() + newDbUserName)
-        .replace('_', '-');
+    return ("create-" + this.resource.getDatabaseName()).replace('_', '-');
```

Comment:
I changed the naming convention here because the db user name isn't always a thing anymore and the database name needs to be unique in the landing zone so that's all we really need